plugins/postcss-is-pseudo-class/.tape.mjs

```diff
@@ -19,11 +19,23 @@ postcssTape(plugin)({
 	},
 	'basic:oncomplex:warning': {
 		message: "supports basic usage with { onComplexSelector: 'warning' }",
-		warnings: 8,
+		warnings: 11,
```

Comment:
Recursion is handled better, so the result has more selectors with `:is` remaining, so more warnings.

The previous version halted transforms earlier.
This is a good thing.